packages/cubejs-testing/test/__snapshots__/smoke-multidb.test.ts.snap

```diff
@@ -1,18 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`multidb query: query 1`] = `
-Array [
-  Object {
-    "Products.name": "apples",
-    "Suppliers.company": "Fruits Inc",
-  },
-  Object {
-    "Products.name": "bananas",
-    "Suppliers.company": "Fruits Inc",
-  },
-  Object {
-    "Products.name": "oranges",
-    "Suppliers.company": "Orchards Inc",
-  },
-]
-`;
+ exports[`multidb query: query 1`] = `
```

Comment:
Not sure why this changed? Can you please `git checkout master -- packages/cubejs-testing/test/__snapshots__/smoke-multidb.test.ts.snap` to revert?